core/camel-core-engine/src/main/docs/modules/eips/pages/resume-strategies.adoc

```diff
@@ -0,0 +1,179 @@
+= Resume Strategies
+:doctitle: Resume Strategies
+:shortname: resume
+:description: Provide strategies to allow consuming data from specific offsets
+:since:
+:supportlevel: Experimental
+
+The resume strategies allow users to implement strategies that point the consumer part of the routes to the last point of consumption. This allows Camel to skip reading and processing data that has already been consumed.
+
+The resume strategies can be used to allow quicker stop and resume operations when consuming large data sources. For instance, imagine a scenario where the file consumer is reading a large file. Without a resume strategy, stopping and starting Camel would cause the consumer in the File component to read all the bytes of the given file at the initial offset (offset 0). The resume strategy allow integrations can point the consumer to the exact offset to resume the operations.
+
+The resume strategies comes in 3 parts:
+
+* A DSL method that marks the route as supporting resume operations and points to an instance of a strategy implementation.
+* A set of core infrastructure that allow integrations to implement different types of strategies
+* Basic strategies implementations that can be extended to implement the specific resume strategies required by the integrations
+
+=== The DSL method
+
+The route needs to use the `resumable()` method followed by a `resumableStrategyRef` to point to an instance of the resume strategy in use. This instance needs to be bound in the Context registry.
```

Comment:
xxxRef is old style, we are adjusting the DSL to be just xxx, so we should rename this

Reply:
Why does it need to be bound, in Java DSL you should IMHO also be able to use the instance object also, resumableStrategy(new MyStrategy())